diff --git a/MacOSX/configure b/MacOSX/configure
index 1ba7b30..d5933cb 100755
--- a/MacOSX/configure
+++ b/MacOSX/configure
@@ -66,7 +66,7 @@ then
 	echo -en $NORMAL
 else
 	echo "Universal Binary build"
-	CFLAGS="$CFLAGS -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.8.sdk -arch i386 -arch x86_64"
+	CFLAGS="$CFLAGS -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -arch x86_64"
 fi
 echo
 
