Index: Drivers/ccid/src/ccid_usb.c
===================================================================
--- Drivers/ccid/src/ccid_usb.c	(revision 6811)
+++ Drivers/ccid/src/ccid_usb.c	(working copy)
@@ -311,6 +311,16 @@
 		}
 	}
 
+#ifdef __APPLE__
+	/* XXX: there seems to be a bug in libusb that makes this specific device not
+	   show up properly for libusb after it is (re)plugged. Giving it a bit more time
+	   makes it show up in libusb list. Not an elegant solution but makes the device
+	   work. */
+	if (!strcmp("Generic USB2.0-CRW", device)) {
+		DEBUG_CRITICAL("Giving Realtek chip some more time...");
+		usleep(900000);
+	}
+#endif
 	cnt = libusb_get_device_list(ctx, &devs);
 	if (cnt < 0)
 	{
